model/src/main/kotlin/config/ScanStorageConfiguration.kt

```diff
@@ -72,9 +72,9 @@ data class PostgresStorageConfiguration(
     val url: String,
 
     /**
-     * The name of the database to use.
+     * The schema used by the database. Default is public.
      */
-    val schema: String,
+    val schema: String = "public",
```

Comment:
Does this imply the risk to overwrite existing data in that default scheme? Wouldn't it be better to create any non-existing scheme, and set the default to something like "ort" here?

Reply:
Will not overwrite data in default scheme, but then the idea of create own ort scheme is very good.
Let me think about it

Reply:
I don't think there's a risk of data loss, ORT is never deleting tables or columns, it only adds missing tables or columns. Scan results are also never overwritten, but IIRC for file archives this might be the case, but I don't see a problem there.
On the other hand it is likely that the role used for adding/reading scan results and file archives does not have permission to create schemas or change schema permissions. And I also think that it's a good idea to use a role without any permissions outside of the schema that should be used by ORT in a production environment.
Therefore I think we should not bother with schema creation, in a testing environment it's fine to use "public" by default, and in a production environment you will want to have more fine grained control over the permission anyway.

Reply:
Changed approach